### PR TITLE
Add exhibit-style browse category landing page 

### LINKED ISSUE #919: Exhibity-style browse detail page
Related to #786

### PR DESCRIPTION
**Pull request by jkeck**
Closes #919 

### Browse Landing Page
browse-landing

### Search With Masthead
search-masthead

### Search Without Masthead
exhibit-masthead

### CODE REVIEW

**Comment by cbeer** [comment]
:+1:

**Comment by ggeisler** [comment]
This is great, @jkeck. I have some styling suggestions for the case when we're showing the browse-specific masthead. I'll list them here and you can either try to integrate them with this pull request or you can ignore for now and I'll do a styling-oriented ticket later to make the updates.

* Make title text smaller. I think it is using a `.jumbotron .h1` style which makes the masthead text too big. I'd go with the normal (non-jumbtron) `h1` font-size of `$font-size-h1` (~36px)
* Apply the text shadow setting we used on the site image masthead to the title and item count
* Remove the border and border-radius that is on the `#exhibit-navbar` 
* Darken the `#exhibit-navbar` to `$gray`
* Lighten the `.navbar-default .navbar-brand` style (site title in navbar) to `$gray-lighter`
*  Lighten the `.navbar-default .navbar-nav > li > a` style (site links in navbar) to `$gray-lighter`

Which should get it looking more like this:

digital_maps___bob_fitch_photography_archive_-_blacklight_and_internal_ ___sites_stanford_spotlight_spec_internal_ _ruby_ _158x50

**Comment by ggeisler** [comment]
Also, let's bump up the `column-gap` for the description text to 3em (currently 1em).

**Comment by jkeck** [comment]
@ggeisler updated styling

updated-search-masthead

**Comment by ggeisler** [comment]
:+1: Looks great!

File: app/views/shared/_exhibit_navbar.html.erb [hunk 1]
```diff
@@ -8,7 +11,7 @@
         <% end %>
       <% end %>
     </ul>
-    <% if !current_exhibit || current_exhibit.searchable? %>
+    <% if !current_exhibit || current_exhibit.searchable? && !current_search_masthead? %>
```
Comment (jkeck) [inline app/views/shared/_exhibit_navbar.html.erb]: I'm going to move this into a helper.

**Pull request merged**